src/sentry/static/sentry/app/views/settings/organizationSecurityAndPrivacy/organizationSecurityAndPrivacy.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import Feature from 'app/components/acl/feature';
+import FeatureDisabled from 'app/components/acl/featureDisabled';
+import {PanelAlert} from 'app/components/panels';
+import {t} from 'app/locale';
+import withOrganization from 'app/utils/withOrganization';
+
+import OrganizationSecurityAndPrivacyContent, {
+  OrganizationSecurityAndPrivacyContentProps,
+} from './organizationSecurityAndPrivacyContent';
+
+const OrganizationSecurityAndPrivacy = ({
+  organization,
+  ...props
+}: OrganizationSecurityAndPrivacyContentProps) => (
```

Comment:
I made a similar comment in #17128 but we don't need to export props like this, instead we can access them via the component

```suggestion
}: React.ComponentProps<OrganizationSecurityAndPrivacy>) => (
```

Reply:
@billyvg  oh now I got it :)